readme.md

```diff
@@ -117,6 +117,39 @@ mixpanel.alias('distinct_id', 'your_alias');
 // callback as the last argument
 mixpanel.track('test', function(err) { if (err) throw err; });
 
+// track multiple events at once
+mixpanel.track_batch({
+    event_list: [
+        {
+            event: 'recent event',
+            properties: {
+                time: new Date(),
+                distinct_id: 'billybob',
+                gender: 'male'
+            }
+        },
+        {
+            event: 'another recent event',
+            properties: {
+                distinct_id: 'billybob',
+                color: 'red'
+            }
+        }
+    ]
+});
+
+// track multiple events at once
+mixpanel.people.batch({
```

Comment:
I think about it, but all the functions of people are in people object, and i don't know how will be better.